addons/binding/org.openhab.binding.kostalinverter/ESH-INF/i18n/kostalinverter_de.properties

```diff
@@ -0,0 +1,33 @@
+# binding
+binding.kostalinverter.name = Kostal Inverter Binding
+binding.kostalinverter.description = Dieses Binding liest die Daten eines Kostal Wechselrichter dessen Weboberfl�che aus.
```

Comment:
"Dieses Binding liest die Daten eines Kostal Wechselrichter aus dessen Weboberfläche aus."

Reply:
I will append the "s" in the case too. "... Kostal Wechselrichters aus ..."